shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -11,6 +11,26 @@
 static const char _kTextAffinityDownstream[] = "TextAffinity.downstream";
 static const char _kTextAffinityUpstream[] = "TextAffinity.upstream";
 
+#pragma mark - TextInputConfiguration Field Names
+static NSString* const _kSecureTextEntry = @"obscureText";
```

Comment:
You shouldn't have the `_` prefix here: https://google.github.io/styleguide/objcguide.html#constants